converters/provider-aws/common.go

```diff
@@ -129,8 +134,26 @@ func ConvertComposedTemplateTags(sourceTemplate xpv1.ComposedTemplate) ([]xpv1.P
 // RegisterAllKnownConverters registers all known converters for provider-aws
 // All future API converters for the community AWS provider must be registered in this function for the correct GVK
 func RegisterAllKnownConverters(r *migration.Registry) {
+	r.RegisterAPIConversionFunctions(apigatewayv2v1alpha1.APIGroupVersionKind,
+		apigatewayv2.APIResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(apigatewayv2v1alpha1.APIMappingGroupVersionKind,
+		apigatewayv2.APIMappingResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(apigatewayv2v1alpha1.DeploymentGroupVersionKind,
+		apigatewayv2.DeploymentResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(apigatewayv2v1alpha1.DomainNameGroupVersionKind,
+		apigatewayv2.DomainNameResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(apigatewayv2v1alpha1.IntegrationGroupVersionKind,
+		apigatewayv2.IntegrationResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(apigatewayv2v1alpha1.RouteGroupVersionKind,
+		apigatewayv2.RouteResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(apigatewayv2v1alpha1.StageGroupVersionKind,
+		apigatewayv2.StageResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(apigatewayv2v1beta1.VPCLinkGroupVersionKind,
+		apigatewayv2.VPCLinkResource, migration.DefaultCompositionConverter(nil, ConvertComposedTemplateTags), DefaultPatchSetsConverter)
 	r.RegisterAPIConversionFunctions(cloudfrontv1alpha1.DistributionGroupVersionKind,
 		cloudfront.DistributionResource, nil, DefaultPatchSetsConverter)
+	r.RegisterAPIConversionFunctions(cloudfrontv1alpha1.ResponseHeadersPolicyGroupVersionKind,
+		cloudfront.ResponseHeadersPolicyResource, nil, DefaultPatchSetsConverter)
```

Comment:
Do we have any reason for not using the default composition converter? As I observed, for another resources, we are using this.